src/test/java/edu/carroll/dogtag/service/LoginServiceImplTest.java

```diff
@@ -0,0 +1,554 @@
+    }
+
+    @Test
+    public void validateUserFailInputSwitchTest() {
+        Login createUser = new Login();
+        createUser.setPassword(password);
+        createUser.setUser(user);
+        createUser.setEmail(email);
+        final LoginForm form = new LoginForm(password, user);
+        assertTrue("Register should return true {}", registerService.register(createUser));
+        assertNotNull("User should be found in the database {}", loginService.findLogin(user));
+        assertFalse("validateUserFailInputSwitchTest: should fail switching user/pass info", loginService.validateUser(form));
+    }
+
+    @Test
+    public void validateUserExistingUserInvalidPasswordTest() {
+        Login createUser = new Login();
+        createUser.setPassword(password);
+        createUser.setUser(user);
+        createUser.setEmail(email);
+        final LoginForm form = new LoginForm(user, password + "extra");
+        assertTrue("Register should return true {}", registerService.register(createUser));
+        assertNotNull("User should be found in the database {}", loginService.findLogin(user));
+        assertFalse("validateUserExistingUserInvalidPasswordTest: should fail using a valid user, invalid pass", loginService.validateUser(form));
+    }
+
+    @Test
+    public void validateUserInvalidUserValidPasswordTest() {
+        Login createUser = new Login();
+        createUser.setPassword(password);
+        createUser.setUser(user);
+        createUser.setEmail(email);
+        final LoginForm form = new LoginForm(user + "not", password);
+        assertTrue("Register should return true {}", registerService.register(createUser));
+        assertNotNull("User should be found in the database {}", loginService.findLogin(user));
+        assertFalse("validateUserExistingUserInvalidPasswordTest: should fail using an invalid user and correct pass", loginService.validateUser(form));
+    }
+
+    @Test
+    public void validateUserInvalidUserInvalidPasswordTest() {
+        Login createUser = new Login();
+        createUser.setPassword(password);
+        createUser.setUser(user);
+        createUser.setEmail(email);
+        final LoginForm form = new LoginForm(user + "not", password + "extra");
+        assertTrue("Register should return true {}", registerService.register(createUser));
+        assertNotNull("User should be found in the database {}", loginService.findLogin(user));
+        assertFalse("validateUserExistingUserInvalidPasswordTest: should fail using a invalid user, invalid pass", loginService.validateUser(form));
+
+    }
+
+    @Test
+    public void validateUserInvalidUserInvalidPasswordTest2() {
+        Login createUser = new Login();
+        createUser.setPassword(password);
+        createUser.setUser(user);
+        createUser.setEmail(email);
+        final LoginForm form = new LoginForm(user + "not", password + "extra");
```

Comment:
I'd rename the test method variables to more descriptive names.  `form` should be `invalidUserForm` if you don't expect it to be found, as it makes the code easier to follow.